src/elm/Pages/Build/View.elm

```diff
@@ -655,32 +645,20 @@ viewLines : FocusLine msg -> Resource -> ResourceID -> LogFocus -> String -> Boo
 viewLines focusLine resource resourceID logFocus decodedLog shiftDown =
     let
         lines =
-            if not <| logEmpty decodedLog then
```

Comment:
we no longer need this code because a decodedLog will never be empty.
if the log were empty, it would be decoded with the helpful message and be safe to render.